streams/src/test/java/org/apache/kafka/streams/state/internals/CachingSessionStoreTest.java

```diff
@@ -234,69 +238,69 @@ public void shouldForwardChangedValuesDuringFlush() {
         final Windowed<Bytes> b = new Windowed<>(keyA, new SessionWindow(1, 2));
         final Windowed<String> aDeserialized = new Windowed<>("a", new SessionWindow(2, 4));
         final Windowed<String> bDeserialized = new Windowed<>("a", new SessionWindow(1, 2));
-        final List<KeyValue<Windowed<String>, Change<String>>> flushed = new ArrayList<>();
```

Comment:
replace `flushed` to reuse existing `CachingKeyValueStoreTest.CacheFlushListenerStub`

Reply:
Nice cleanup!